File: pb/src/main/java/org/sqlbuilder/sl/PbRoleResolver.java
package org.sqlbuilder.sl;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;
import org.sqlbuilder2.ser.Pair;
import org.sqlbuilder2.ser.Triplet;

import java.io.File;
import java.io.IOException;

public class PbRoleResolver extends Resolver<Pair<String,String>, Triplet<Integer,Integer,Integer>>
{
	public PbRoleResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: pm/src/main/java/org/sqlbuilder/pm/PbRoleResolver.kt
package org.sqlbuilder.pm

import org.sqlbuilder.common.DeSerialize.deserialize
import org.sqlbuilder.common.Resolver
import java.io.File

class PbRoleResolver(ser: String) : Resolver<Pair<String, String>, Pair<Int, Int>>(deserialize<Map<Pair<String, String>, Pair<Int, Int>>>(File(ser)))
